libafl/src/events/centralized.rs

```diff
@@ -47,19 +45,18 @@ pub(crate) const _LLMP_TAG_TO_MAIN: Tag = Tag(0x3453453);
 
 /// A wrapper manager to implement a main-secondary architecture with another broker
 #[derive(Debug)]
-pub struct CentralizedEventManager<EM, EMH, I, S, SHM, SP> {
+pub struct CentralizedEventManager<EM, I, S, SHM, SP> {
     inner: EM,
     /// The centralized LLMP client for inter process communication
     client: LlmpClient<SHM, SP>,
     #[cfg(feature = "llmp_compression")]
     compressor: GzipCompressor,
     time_ref: Option<Handle<TimeObserver>>,
-    hooks: EMH,
```

Comment:
Can we get dynamic hooks back now?

Reply:
CustomBuf? no that one is bad.
but instead i can impl EventManagerHooks for Vec